samples/AspNetCore/ControllerSample/Controllers/SampleController.cs

```diff
@@ -80,5 +80,20 @@ public async Task<ActionResult<Account>> Withdraw(Transaction transaction, [From
             await state.SaveAsync();
             return state.Value;
         }
+
+        /// <summary>
+        /// Method for throwing a BadRequest exception which will cause Dapr sidecar to throw an RpcException
+        /// </summary>
+        /// <param name="transaction">Transaction info.</param>
+        /// <param name="daprClient">State client to interact with Dapr runtime.</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
```

Comment:
I don't think we should continue the practice of placing doc comments on every method in samples. There's a lot of boilerplate here that doesn't matter to anyone.

I'd suggest a normal comment that explains why this is here (what you have in the summary).

Reply:
agree....the other stuff does not really matter in this case

Reply:
The repo was set up to enforce full doc comments on all samples and tests - I got rid of that 😎